generator.json

```diff
@@ -1817,16 +2021,16 @@
                     "ISequentialStream": "Silk.NET.Core.Win32Extras.ISequentialStream",
                     "IPersist": "Silk.NET.Core.Win32Extras.IPersist",
                     "IPersistStream": "Silk.NET.Core.Win32Extras.IPersistStream",
-                    "IPropertyBag2": "Silk.NET.Core.Native.IUnknown",
+                    "IPropertyBag2": "Silk.NET.Core.Win32Extras.IPropertyBag2",
                     "IEnumUnknown": "Silk.NET.Core.Win32Extras.IEnumUnknown",
                     "IEnumString": "Silk.NET.Core.Win32Extras.IEnumString",
                     "tagSAFEARRAY": "Silk.NET.Core.Win32Extras.SafeArray",
                     "tagSTATSTG": "Silk.NET.Core.Win32Extras.STATSTG",
-                    "tagPROPVARIANT": "void",
-                    "tagPROPBAG2": "void",
-                    "HPALETTE__": "void",
-                    "HBITMAP__": "void",
-                    "HICON__": "void",
+                    "tagPROPVARIANT": "Silk.NET.Core.Win32Extras.PropVariant",
+                    "tagPROPBAG2": "Silk.NET.Core.Win32Extras.PropBag2",
+                    "HPALETTE__": "IntPtr",
+                    "HBITMAP__": "IntPtr",
+                    "HICON__": "IntPtr",
```

Comment:
What's the reason for this change? A `HICON__` is the opaque struct pointed to by a `HICON` (that is, a `HICON` is a `HICON__*` where `HICON__` is empty). We have sometimes used `IntPtr` in place of pointers to types like these, but not in place of the opaque pointee. I appreciate that `void*` isn't the best dev ex, but I'm not sure why this is the change.

Can we revert this?

Reply:
Done.